src/ResourceManagement/ServerManagement/ServerManagement/Microsoft.Azure.Management.ServerManagement.nuget.proj

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup>
+    <!--
+    Microsoft.Azure.Management.ServerManagement
+    -->
+    <SdkNuGetPackage Include="Microsoft.Azure.Management.ServerManagement">
+      <PackageVersion>1.0.3</PackageVersion>
```

Comment:
This shouldn't be marked as preview; there definitely won't be breaking changes; the actual API hasn't changed in months.